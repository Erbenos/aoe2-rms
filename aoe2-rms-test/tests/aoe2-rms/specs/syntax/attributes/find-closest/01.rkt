#lang aoe2-rms
; Give each player a fishing ship on the closest free water tile

<OBJECTS-GENERATION>
(create-object 'FISHING_SHIP
               (set-place-for-every-player)
               (ignore-terrain-restrictions)
               (terrain-to-place-on 'WATER)
               (find-closest))
