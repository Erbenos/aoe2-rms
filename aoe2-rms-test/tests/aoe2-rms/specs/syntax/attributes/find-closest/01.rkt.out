<OBJECTS_GENERATION>
create_object FISHING_SHIP {
  set_place_for_every_player
  ignore_terrain_restrictions
  terrain_to_place_on WATER
  find_closest
}
